.github/workflows/directory_workflow.yml

```diff
@@ -1,46 +0,0 @@
-name: Directory/Filename Formatter workflow
```

Comment:
Was this completely removed and not being used anymore?

Reply:
This only adds complexity and causes annoyances during
a push and pull request. Removed. Doing this
manually is the best way. Last time i fetch and pulled to my local repository, it destroyed all my changes

Reply:
I do think it's a useful tool but I think that should be done manually than adding it into an automation. We can direct contributors to the script.